.storybook/preview.js

```diff
@@ -1,6 +1,30 @@
 import React from "react";
 import wonderBlocksTheme from "./wonder-blocks-theme.js";
 
+const wbViewports = {
```

Comment:
**explanation:** Trying to use our defined breakpoints... see https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2099970518/Layout+Breakpoints

Note: I'm going to add this link as a comment in the code.